README.md

```diff
@@ -85,4 +85,16 @@ The current `package-storage` repository has a few branches. This is a quick sum
 * staging: Packages for epr-staging.elastic.co
 * snapshot: Packages for epr-snapshot.elastic.co
 * experimental: Packages for epr-experimental.elastic.co. These packages are served by Kibana 7.8 and will disappear in the future. No updates should happen to this branch.
-* master: Contains docs and comment scripts for the distribution branches.
\ No newline at end of file
+* master: Contains docs and comment scripts for the distribution branches.
+
+# Tags
+
+At the time of the Elastic Stack release, a tag off the `production` branch is created the Elastic Stack release number. This will allow environments like Kibana tests to use a specific "known" tag for their testing. An additional benefit is that in the future if users potentially run the registry on prem, these tags can be used to have the on prem registry aligned with the stack version instead of constantly pulling the most recent version.
```

Comment:
```suggestion
At the time of the Elastic Stack release, a tag off the `production` branch is created with an Elastic Stack release version. This will allow environments like Kibana tests to use a specific "known" tag for their testing. An additional benefit is that in the future if users potentially run the registry on premise, these tags can be used to have the on premise registry aligned with the Elastic Stack version instead of constantly pulling the most recent version.
```